clab/config_test.go

```diff
@@ -471,22 +482,6 @@ func TestLabelsInit(t *testing.T) {
 	}
 }
 
-func TestVerifyRootNetnsInterfaceUniqueness(t *testing.T) {
-	opts := []ClabOption{
-		WithTopoFile("test_data/topo7-dup-rootnetns.yml", ""),
-	}
-	c, err := NewContainerLab(opts...)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	err = c.verifyRootNetnsInterfaceUniqueness()
-	if err == nil {
-		t.Fatalf("expected duplicate rootns links error")
-	}
-	t.Logf("error: %v", err)
-}
-
```

Comment:
@steiler this test was removed, as the underpinning function `verifyRootNetnsInterfaceUniqueness` was removed. But the check disappeared, you can try deploying the lab test_data/topo7-dup-rootnetns.yml and it fails.

We have to fix this

Reply:
Hhmmm shit... that is two bridges that live in the root net ns. That was not on my radar.
The check disappeared, because my assumption now was, that we have the maps of links per node and that we would thereby figure out is there is a dup. but two different bridges also live in the host netns, thats correct.
I'll come up with something